.github/workflows/build-prod-container.yml

```diff
@@ -141,7 +155,7 @@ jobs:
             then
               GITHUB_TAG=${GITHUB_REF##*/}
             else
-              GITHUB_TAG="latest"
+              GITHUB_TAG="latest-dev"
```

Comment:
If there's no tags on the current commit, we set it to `latest-dev` instead of `latest` to avoid confusion; however, since we only push images if they are part of a release, I don't expect this to be present in the registry at all.